pkg/cmd/cli/describe/printer.go

```diff
@@ -34,7 +34,7 @@ var (
 	imageColumns            = []string{"NAME", "DOCKER REF"}
 	imageStreamTagColumns   = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
 	imageStreamImageColumns = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
-	imageStreamColumns      = []string{"NAME", "DOCKER REPO", "TAGS", "UPDATED"}
+	imageStreamColumns      = []string{"NAME", "TAGS", "UPDATED"}
```

Comment:
hrm, "not every image can be docker pulled" - example? (I can think of online where the registry is not exposed with route). 

just to double-check, we are exposing this information (docker pull spec) when you get the "oc get ist/foo:bar" right?